install/kubernetes/cilium/templates/_hubble-generate-certs-job-spec.tpl

```diff
@@ -17,61 +17,47 @@ spec:
           {{/* Because this is executed as a job, we pass the values as command line args instead of via config map,
                 this allows users to inspect the values used in past runs by inspecting the completed pod */ -}}
           args:
+            - "--cilium-namespace={{ .Release.Namespace }}"
```

Comment:
I did not notice that certgen can use this for hubble already, most likely just missed it.

Reply:
Ah yes, this is very recent :) https://github.com/cilium/certgen/pull/10